api/src/main/java/uk/gov/dhsc/htbhf/dwp/service/EligibilityService.java

```diff
@@ -55,13 +55,13 @@ public EligibilityResponse checkEligibility(DWPEligibilityRequest eligibilityReq
         String nino = eligibilityRequest.getPerson().getNino();
         Optional<UCHousehold> ucHousehold = ucHouseholdRepository.findHouseholdByAdultWithNino(nino);
         if (ucHousehold.isPresent()) {
-            log.debug("Matched UC household: {}", ucHousehold.get().getHouseholdIdentifier());
+            log.debug("Matched UC household: {}", ucHousehold.get().getId());
```

Comment:
Does this mean we are logging out DEBUG level output in Production? If not, then I don't see that we need to make this change.

Reply:
yes, we're logging at debug in production currently. Regardless, I think we should ensure we don't log personally identifiable information.